cypress/integration/basic_commands.spec.ts

```diff
@@ -176,6 +176,11 @@ describe('Basic Commands Test', () => {
     const username = uuid();
     const password = uuid();
     cy.get('[data-login-button]').click();
+    cy.get('[data-signup-birthdate]').type(
+      getISODate(
+        addSubtractDaysToDate(now, {years: -18}),
+      ),
+    );
```

Comment:
I noticed the function `addSubtractDaysToDate` only adds/substracts days by a date given, so you need to calculate the age of this way:
```suggestion
    now.setFullYear(now.getFullYear() - 18);
    cy.get('[data-signup-birthdate]').type(getISODate(now));
```

Reply:
Thanks! I did the suggested changes.